working/manual_changes/2021/2021-03-29.sql

```diff
@@ -0,0 +1,29 @@
+DO $_$
+BEGIN
+	PERFORM vocabulary_pack.AddNewRelationship(
+	pRelationship_name			=>'Concepts with reference to variant',
```

Comment:
Why is it plural?
Why “concepts” is even mentioned? It’s all about concepts.

Reply:
Changed